docs/route.md

```diff
@@ -182,6 +182,7 @@ Argument        | Type                                     | Description
 --------------- | ---------------------------------------- | ---
 `args`          | `Object`                                 | The [routing parameters](#routing-parameters)
 `requestedPath` | `String`                                 | The router path requested by the last routing action, including interpolated routing parameter values, but without the prefix. When `onmatch` is called, the resolution for this path is not complete and `m.route.get()` still returns the previous path.
+`route`         | `String`                                 | The router path requested by the last routing action, excluding interpolated routing parameter values
```

Comment:
Don't forget to also update the `` `routeResolver.onmatch(args, requestedPath)` `` bit above to include this new extra parameter.

Reply:
done